src/Form/RedirectForm.php

```diff
@@ -76,7 +76,7 @@ public function form(array $form, FormStateInterface $form_state) {
         '#type' => 'language_select',
         '#title' => t('Language'),
         '#languages' => Language::STATE_ALL,
```

Comment:
This adds the element to the form, but `#access` on the widget is false, even for UID 1. Digging into that now.